### DISCUSSION TITLE
Azure Open AI

### DISCUSSION THREAD

**Discussion by don-straka** [brianpetro/obsidian-smart-connections#577] [Q&A]
Is Azure Open AI now an option for Smart Chart with version 2.1? That is, can I use my private Azure Open AI API Key to power the chat function?

**Comment by jullimullins** [comment]
I would like to get help on this too. I have got TextGenerator working with Azure OpenAI api. In TextGenerator you need to give Api key, Endpoint, Deployment name, model and Api version but Smart Connection you need to give model name, protocol, hostname and api key. I checked the console but that didnt give me any hints.
